chainntnfs/btcdnotify/btcd.go

```diff
@@ -481,6 +483,13 @@ func (b *BtcdNotifier) attemptHistoricalDispatch(msg *confirmationsNotification,
 	// Otherwise, the transaction has only been *partially* confirmed, so
 	// we need to insert it into the confirmation heap.
 	confsLeft := msg.numConfirmations - uint32(tx.Confirmations)
+
+	// Account for the possibility that the transaction might've been included
+	// in a block that occured since currentHeight's last update
+	if int32(block.Height) > currentHeight {
```

Comment:
Can we skip this and just do `confHeight := block.Height + msg.numConfirmations - 1`? Then the method wouldn't even need `currentHeight` as a parameter.

Reply:
Nice, I like way better too. Have updated it now.